packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -149,6 +149,12 @@ module.exports = {
         // match the requirements. When no loader matches it will fall
         // back to the "file" loader at the end of the loader list.
         oneOf: [
+          // This is included by default, but when we add .json to eslint-loader
+          // test regex the compiler fails to recognize json files.
```

Comment:
This looks a bit weird. Can we figure out why this happens and if there's a nicer way to do it?

Reply:
https://github.com/webpack-contrib/json-loader/issues/44 which links to https://webpack.js.org/guides/migrating/#json-loader-is-not-required-anymore

>When no loader has been configured for a JSON file, webpack will automatically try to load the JSON file with the json-loader.

I think adding json test in eslint-loader counts as `loader has been configured for a JSON file`